packages/react/src/components/Tooltip/Tooltip.tsx

```diff
@@ -238,6 +242,7 @@ export default class Tooltip extends AutoControlledComponent<TooltipProps, Toolt
         pointerTargetRef={this.pointerTargetRef}
         align={align}
         position={position}
+        enabled={this.state.open}
```

Comment:
```diff
    const { align, position, target } = this.props
+    const { open } = this.state
```

```suggestion
        enabled={open}
```

_nit:_ to aligned let's use destruction